internal/declarative/v2/reconciler.go

```diff
@@ -368,6 +368,20 @@ func (r *Reconciler) pruneDiff(
 	return renderer.RemovePrerequisites(ctx, obj)
 }
 
+func pruneKymaSystem(diff []*resource.Info) []*resource.Info {
+	for i, info := range diff {
+		obj := info.Object.(client.Object)
+		if obj.GetObjectKind().GroupVersionKind().Kind != "Namespace" {
+			continue
+		}
+		if obj.GetName() != "kyma-system" {
```

Comment:
this magic `kyma-system` it's better to be extracted as a const, and referenced in the test, so that we get the purpose here more clearly

Reply:
Const is fine, but it should **not** be referenced in tests, because if it gets changed accidentally the test won't notice.

Reply:
after reading your refactored comment, it's fine, what I originally meant for this reference is, to reference this const when you create your input for the test, but of cause not reference it as the expected value for the test. I think that should be clear.